.github/workflows/stale-issues-bot.yaml

```diff
@@ -0,0 +1,23 @@
+name: "stale issues bot"
+on:
+  schedule:
+  - cron: "0 7 * * *"
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale-bot:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v8
+      with:
+        operations-per-run: 30
+        days-before-stale: 90
+        days-before-close: 30
+        stale-issue-label: stale
+        stale-pr-label: stale
+        exempt-issue-labels: no-autoclose
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'The Stale Bot detected no activity in the past 90 days. This issue will be automatically closed if no further activity occurs in the next 30 days.'
```

Comment:
I would be a bit more friendly to people here, and explain why we (have to) take such measures.